StreamerBotPlugin/Commands/ActionAdjustmentCommand.cs

```diff
@@ -34,27 +36,41 @@ public class ActionAdjustmentCommand : PluginDynamicAdjustment
         private readonly HttpService _httpService;
         private Action[] _actions;
 
+        private static readonly ConcurrentDictionary<String, Int32> _ActionAdjustmentValues = new ConcurrentDictionary<String, Int32>();
+
         public ActionAdjustmentCommand() : base("Adjustment Action", "Choose and execute your actions with adjustments (Reset sets it to 0)", "Adjustment Action", true)
         {
             this._httpService = HttpService.Instance;
-            this._actions = this._httpService.GetActions()?.Actions ?? Array.Empty<Action>();
+            this._setActions().Wait();
             this.MakeProfileAction("tree");
         }
 
+        private async Task<Action[]> _setActions() =>
+            this._actions = (await this._httpService.GetActions()).Actions;
+
+
         protected override PluginProfileActionData GetProfileActionData()
         {
-            this._actions = this._httpService.GetActions()?.Actions ?? Array.Empty<Action>();
+            this._setActions().Wait();
```

Comment:
This is one good example what I mean in `1.`

Reply:
This makes sense. 
To me it made no difference sprinkling `.Result()` and `.Wait()` everywhere, but I can see why you would have wanted the service to encapsulate, so it never left the boundaries of that type. 

My revision will roll this back to how it originally was.